### PR TITLE
Aqua post config 

### PR DESCRIPTION
**Pull request by patricksimonian**
Created the aqua post config playbook.

In its MVP form it:
- creates the oidc config
- creates the aqua scanner user
- enables enforcers

### CODE REVIEW

File: apps/aqua-post-config/playbook/group_vars/all.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+aqua_url: https://aqua.apps.clab.devops.gov.bc.ca
```
Comment (j-pye) [inline apps/aqua-post-config/playbook/group_vars/all.yaml:1]: Just a thought but couldn't this also be an ENV? Extra vars `-e` will take priority if set but if you use an ENV it's one less thing to pass in for k8s Job.

Also, up to you but I see you're missing a bunch of new lines at the end of your files 🕵🏻‍♂️

**Review by j-pye — APPROVED** [review]
Left a suggestion but not a requirement. Looks good otherwise so approving! 🚀

File: apps/aqua-post-config/playbook/tasks/enable_oidc.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      client_id: "{{ aqua_sso_client }}"
+      enabled: true
+      idp_url: "{{ aqua_sso_url }}"
+      provider_name: "sso"
+      redirect_url: "{{ redirect_uri }}"
+      role_mapping: {Administrator: ["aqua_admins"]}
+      scopes: ["email", "groups"]
+      secret: "{{ aqua_sso_client_secret }}"
+      user: "email"
+      user_role: "groups"
```
Comment (patricksimonian) [inline apps/aqua-post-config/playbook/tasks/enable_oidc.yaml:27]:
```suggestion
      user_role: "groups"
      
```

File: apps/aqua-post-config/playbook/tasks/login.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    body_format: json
+    return_content: true
+    body:
+      id: "{{ aqua_admin }}"
+      password: "{{ aqua_admin_password }}"
+  failed_when: result.status != 200
+
+- name: Setting Token
+  set_fact:
+    auth_token: "{{ result.json.token }}"
```
Comment (patricksimonian) [inline apps/aqua-post-config/playbook/tasks/login.yaml:16]:
```suggestion
    auth_token: "{{ result.json.token }}"
    
```

File: apps/aqua-post-config/playbook/group_vars/all.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+aqua_admin: administrator
+aqua_sso_url: "https://oidc.gov.bc.ca/auth/realms/8gyaubgq"
```
Comment (patricksimonian) [inline apps/aqua-post-config/playbook/group_vars/all.yaml:3]:
```suggestion
aqua_sso_url: "https://oidc.gov.bc.ca/auth/realms/8gyaubgq"

```

File: apps/aqua-post-config/bin/entrypoint.sh [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+exec tini -- "${@}"
```
Comment (patricksimonian) [inline apps/aqua-post-config/bin/entrypoint.sh:3]:
```suggestion
exec tini -- "${@}"

```

**Pull request merged**